src/gui/processing/qgsprocessingfieldmapwidgetwrapper.cpp

```diff
@@ -133,12 +133,12 @@ void QgsProcessingFieldMapPanelWidget::setValue( const QVariant &value )
   {
     const QVariantMap map = field.toMap();
     QgsField f( map.value( QStringLiteral( "name" ) ).toString(),
-                static_cast< QVariant::Type >( map.value( QStringLiteral( "type" ), QVariant::Invalid ).toInt() ),
-                map.value( QStringLiteral( "type_name" ), QVariant::typeToName( static_cast< QVariant::Type >( map.value( QStringLiteral( "type" ), QVariant::Invalid ).toInt() ) ) ).toString(),
+                static_cast< QMetaType::Type >( map.value( QStringLiteral( "type" ), QgsVariantUtils::createVariant( QMetaType::Type::UnknownType ) ).toInt() ),
```

Comment:
Something has gone wrong here too -- there should be no createVariant call here.

Reply:
same comment as above